cg/services/order_validation_service/workflows/tomte/validation/inter_field/utils.py

```diff
@@ -95,3 +96,13 @@ def is_father_sex_invalid(child: TomteSample, case: TomteCase) -> bool:
 
 def create_father_sex_error(case: TomteCase, sample: TomteSample) -> InvalidFatherSexError:
     return InvalidFatherSexError(sample_name=sample.name, case_name=case.name)
+
+
+def get_mother_sex_errors(case: TomteCase) -> list[InvalidMotherSexError]:
+    errors = []
+    children: list[TomteSample] = case.get_samples_with_mother()
+    for child in children:
+        if is_father_sex_invalid(child=child, case=case):
+            error = create_father_sex_error(case=case, sample=child)
```

Comment:
These seem wrong to me

Reply:
Good catch https://github.com/Clinical-Genomics/cg/pull/3490/commits/e5bff0f2ce4e33651a64c813da209cdd6c861afa